locust/user/wait_time.py

```diff
@@ -34,14 +34,13 @@ def constant_pacing(wait_time):
     execution equal to the time specified by the wait_time argument. 
     
     In the following example the task will always be executed once every second, no matter 
-    the task execution time::
+    the task execution time:
```

Comment:
The double colon is actually not a typo (though I can definitely see why one would think so 😄). It's ReStructuredText syntax.

Screenshot of API docs without double `:`:

![image](https://user-images.githubusercontent.com/54217/82319201-70542a00-99d1-11ea-92b8-576d3c0816ab.png)